index.bs

```diff
@@ -5693,7 +5695,8 @@ interface VTTCue : TextTrackCue {
  interpreted as a time in seconds.</p></li>
 
  <li><p>Let |cue|'s <a>text track cue end time</a> be the value of the |endTime| argument,
- interpreted as a time in seconds.</p></li>
+ interpreted as a time in seconds or positive Infinity for an <a>unbounded text track
```

Comment:
Like in HTML, you'll need to check if the argument is -Infinity of NaN and throw a TypeError. That's because the `endTime` setter defined in HTML isn't called here, rather it's directly setting the internal "text track cue end time".

Reply:
Interesting. I'd assumed that this behaviour would be inherited from `TextTrackCue.endTime` in HTML. Suitable wording added. Thanks.

Reply:
Ah, so it's not the `endTime` setter that needs this, that is indeed inherited (in a sense) from `TextTrackCue`. It's the `VTTCue` constructor that needs to check for -Infinity and NaN.

Reply:
Removed `endTime` setter wording and added equivalent check to step 3 of the `VTTCue` constructor